bodhi/server/models.py

```diff
@@ -1649,13 +1653,14 @@ def get_latest(self):
                 break
         return latest
 
-    def get_changelog(self, timelimit=0):
+    def get_changelog(self, timelimit=0, lastupdate=False):
```

Comment:
Should we default this to `True` since the 2 calls we have are passing `True` ?

Reply:
The `False` choice has been made to preserve the actual behavior of the function. Also setting `lastupdate` to `True` will overwrite the value from `timelimit`, so I think it's safer to let it `False` by default.